connections/tests/factories.py

```diff
@@ -24,50 +25,56 @@ class ApartmentFactory(factory.Factory):
     class Meta:
         model = ApartmentTest
 
+    project_id = fuzzy.FuzzyInteger(0, 9999999999)
+    project_uuid = fuzzy.FuzzyText()
+
     housing_company = fuzzy.FuzzyText()
-    holding_type = fuzzy.FuzzyText()
+    holding_type = "Asumisoikeushuoneisto"
     street_address = fuzzy.FuzzyText()
     postal_code = fuzzy.FuzzyText()
     city = fuzzy.FuzzyText()
     district = fuzzy.FuzzyText()
     realty_id = fuzzy.FuzzyText()
-    construction_year = fuzzy.FuzzyText()
+    construction_year = fuzzy.FuzzyInteger(2000, 3000)
     new_development_status = fuzzy.FuzzyText()
     new_housing = True
     apartment_count = fuzzy.FuzzyInteger(0, 9999999999)
+    parkingplace_count = fuzzy.FuzzyInteger(0, 9999999999)
 
     has_elevator = True
     has_sauna = True
-    construction_material = fuzzy.FuzzyText()
+    construction_materials = factory.List([fuzzy.FuzzyText() for _ in range(2)])
     roof_material = fuzzy.FuzzyText()
-    heating = fuzzy.FuzzyText()
+    heating_options = factory.List([fuzzy.FuzzyText() for _ in range(2)])
     energy_class = fuzzy.FuzzyText()
-    site_area = fuzzy.FuzzyDecimal(0, 9999999999)
+    site_area = fuzzy.FuzzyFloat(0, 9999999999)
```

Comment:
`site_area` is defined as `Long` in the `Apartment` document class. That means `FuzzyInteger` should be used here, right?

Reply:
Hmm.. Yea now that I think about, this field should be defined as `Float` because there can be areas like `32.5m2`. Atm the field is defined as `Long` in the test ES as well. I'll ask about it.